docs/Makefile

```diff
@@ -56,7 +56,6 @@ clean:
 	-rm -rf $(PNGs)
 
 html: openapi latexpdf
-	cp $(BUILDDIR)/latex/inmanta.pdf $(SOURCEDIR)
```

Comment:
I think this has to stay, even if I'm not sure why?

@arnaudsjs do you know why this is?

Reply:
Yes. It has to stay. A PDF version of the documentation is included in the HTML version of the documentation. This command moves the PDF from the output directory of the PDF build to the source directory of the HTML build.